validator/validator-main.protoascii

```diff
@@ -4497,6 +4498,7 @@ tags: {
   attrs: { name: "[disabled]" }
   attrs: { name: "[maxlength]" }
   attrs: { name: "[minlength]" }
+  attrs: { name: "[pattern]" }
```

Comment:
@honeybadgerdontcare A new attribute for `<textarea>`.

Reply:
LGTM, is documentation coming in a different PR?

Reply:
Added some documentation to amp-form.md.